crm_request/models/request.py

```diff
@@ -168,6 +169,14 @@ def message_new(self, msg, custom_values=None):
                 defaults['partner_id'] = partner.id
                 defaults['language'] = partner.lang
 
+        # Check here if the date of the mail is during a holiday
+        mail_date = fields.Date.to_string(
+            fields.Date.from_string(msg.get('date')))
```

Comment:
Is this double conversion needed? Is it changing the date format? Otherwise I think we could just use directly `msg.get('date')`